cynic/src/integrations/chrono.rs

```diff
@@ -25,6 +25,26 @@ impl Scalar for NaiveDate {
 
 crate::impl_serializable_argument_for_scalar!(NaiveDate);
 
+impl Scalar for NaiveTime {
+    fn decode(value: &serde_json::Value) -> Result<Self, DecodeError> {
+        match value {
+            serde_json::Value::String(s) => {
+                Ok(NaiveTime::parse_from_str(s, "%H:%M").map_err(chrono_decode_error)?)
+            }
+            _ => Err(DecodeError::IncorrectType(
+                "String".to_string(),
+                value.to_string(),
+            )),
+        }
+    }
+
+    fn encode(&self) -> Result<serde_json::Value, SerializeError> {
+        Ok(serde_json::Value::String(self.format("%H:%M").to_string()))
```

Comment:
Imagine we might want seconds in here as well - think if we just use `self.to_string()` that'll do the right thing?